src/main/java/ai/deepcode/javaclient/core/AnalysisDataBase.java

```diff
@@ -559,7 +559,7 @@ private List<String> checkBundle(
     @NotNull String bundleId
   ) throws TokenInvalid401Exception, BundleIdExpire404Exception, ApiCallNotSucceedException {
     CreateBundleResponse checkBundleResponse =
-      restApi.checkBundle(deepCodeParams.getSessionToken(), bundleId);
+      restApi.checkBundle(deepCodeParams.getSessionToken(), deepCodeParams.getOrgDisplayName(), bundleId);
```

Comment:
Repeating, just in case: This can be empty - is that a problem?

Reply:
not a problem. empty string is totally fine